comprl/server/managers.py

```diff
@@ -211,6 +214,38 @@ def disconnect_all(self, reason: str) -> None:
         for player in self.connected_players.values():
             player.disconnect(reason)
 
+    def get_matchmaking_parameters(self, user_id: int) -> tuple[float, float]:
+        """
+        Retrieves the matchmaking parameters of a user based on their ID.
+
+        Args:
+            user_id (int): The ID of the user.
+
+        Returns:
+            tuple[float, float]: The mu and sigma values of the user.
+        """
+        return UserData(ConfigProvider.get("user_data")).get_matchmaking_parameters(
+            user_id
+        )
+
+    def update_matchmaking_parameters(self, user_id: int, m: float, s: float) -> None:
+        """
+        Updates the matchmaking parameters of a user based on their ID.
+
+        Args:
+            user_id (int): The ID of the user.
+            m (float): The new mu value of the user.
+            s (float): The new sigma value of the user.
+        """
+        UserData(ConfigProvider.get("user_data")).set_matchmaking_parameters(
+            user_id, m, s
+        )
+
```

Comment:
I personally feel like the names `m` and `s` are a bit confusing. I think `new_mu` and `new_sigma` would be more fitting here.

Reply:
I can agree and changed them, thx